package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "cypress:open": "cypress open"
```

Comment:
To nie jest Ci za bardzo potrzebne, bo równie dobrze możesz odpalić Cypressa już gotową komendą `npx cypress open`, która działa identycznie.

Reply:
obczaję to 🗡️